doc/classes/EditorSettings.xml

```diff
@@ -1,31 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorSettings" inherits="Resource" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Object that holds the project-independent editor settings.
```

Comment:
You lost existing docs, should be brought back (best to reset the file and run doctool).